lib/Doctrine/Persistence/Mapping/Driver/FileDriver.php

```diff
@@ -132,10 +134,10 @@ public function getAllClassNames()
             return (array) $this->locator->getAllClassNames($this->globalBasename);
         }
 
-        return array_unique(array_merge(
+        return array_values(array_unique(array_merge(
```

Comment:
What's the reason for adding `array_values` here? I thought `array_unique` renumbers arrays?

Reply:
It doesn't:

```
>>> array_unique([2 => 3])
=> [
     2 => 3,
   ]
```

Reply:
Wow. I see.